packages_rs/nextclade-web/src/components/Results/ColumnAaMotifs.tsx

```diff
@@ -0,0 +1,349 @@
+import { Tooltip } from 'src/components/Results/Tooltip'
+import { useTranslationSafe, useTranslationSafe as useTranslation } from 'src/helpers/useTranslationSafe'
+import { TableSlim, TableSlimWithBorders } from 'src/components/Common/TableSlim'
+import { InsertedFragmentTruncated } from 'src/components/Results/ListOfInsertions'
+import { geneMapAtom } from 'src/state/results.state'
+
+export interface ColumnAaMotifsProps {
+  analysisResult: AnalysisResult
+  motifDesc: AaMotifsDesc
+}
+
+export function ColumnAaMotifs({ analysisResult, motifDesc }: ColumnAaMotifsProps) {
+  const { t } = useTranslationSafe()
+  const [showTooltip, setShowTooltip] = useState(false)
+  const onMouseEnter = useCallback(() => setShowTooltip(true), [])
+  const onMouseLeave = useCallback(() => setShowTooltip(false), [])
+
+  const { index, seqName, aaMotifsChanges } = analysisResult
+
+  const id = useMemo(
+    () =>
+      getSafeId('col-aa-motif', {
+        index,
+        seqName,
+        name: motifDesc.name,
+      }),
+    [index, motifDesc.name, seqName],
+  )
+
+  const motifs = get(aaMotifsChanges, motifDesc.name)
+
+  const columnValue = useMemo(() => {
+    if (!motifs) {
+      return null
+    }
+    return (
+      <span>
+        <ColoredText>{motifs.total}</ColoredText>
+        <TextNarrow>
+          {'('}
+          <ColoredText $color={'#487921'}>+{motifs.gained.length}</ColoredText>
+          <ColoredText $color={'#7f0d0d'}>-{motifs.lost.length}</ColoredText>
+          {')'}
+        </TextNarrow>
+      </span>
+    )
+  }, [motifs])
+
+  const gained = useMemo(
+    () =>
+      motifs.gained.length > 0 && (
+        <div>
+          <ColoredH6 $color={'#487921'} className="mb-0">
+            {t('Gained: {{gained}}', { gained: motifs.gained.length })}
+          </ColoredH6>
+          <p className="my-0">
+            <small>{t('Motifs which are not present in reference sequence, but appeared in query sequence')}</small>
+          </p>
+          <ListOfAaMotifMutations motifs={motifs.gained} />
+        </div>
+      ),
+    [motifs.gained, t],
+  )
+
+  const lost = useMemo(
+    () =>
+      motifs.lost.length > 0 && (
+        <div>
+          <ColoredH6 $color={'#7f0d0d'} className="mb-0">
+            {t('Lost: {{lost}}', { lost: motifs.lost.length })}
+          </ColoredH6>
+          <p className="my-0">
+            <small>{t('Motifs which are present in reference sequence, but disappeared in query sequence')}</small>
+          </p>
+          <ListOfAaMotifMutations motifs={motifs.lost} />
+        </div>
+      ),
+    [motifs.lost, t],
+  )
+
+  const preserved = useMemo(
+    () =>
+      motifs.preserved.length > 0 && (
+        <div>
+          <ColoredH6 className="mb-0">
+            {t('Preserved: {{preserved}}', { preserved: motifs.preserved.length })}
+          </ColoredH6>
+          <p className="my-0">
+            <small>{t('Motifs carried from reference sequence (sometimes mutated)')}</small>
```

Comment:
Comment maybe 
"Present in reference and query (sometimes mutated, showing query motif)"

Reply:
It shows `ref -> qry` when mutated. If it shows only one, then it is not. This is why I splitted up mutated into a separate table initially.